src/mtcp/mtcp_restart.c

```diff
@@ -104,11 +104,13 @@ typedef struct RestoreInfo {
   VA vvarStart;
   VA vvarEnd;
   fnptr_t post_restart;
+  fnptr_t post_restart_debug;
```

Comment:
Please see my latest PR #564. Apparently, we had missed this in the review of the patch that went into 2.5.